kvirt/defaults.py

```diff
@@ -18,14 +18,14 @@
 START = True
 TUNNEL = False
 TEMPLATES = {'arch': 'https://linuximages.de/openstack/arch/arch-openstack-LATEST-image-bootstrap.qcow2',
-             'atomic': 'https://download.fedoraproject.org/pub/alt/atomic/stable/Fedora-Atomic-25-20170512.2/CloudImages/x86_64/images/Fedora-Atomic-25-20170512.2.x86_64.qcow2',
+             'atomic': 'https://download.fedoraproject.org/pub/alt/atomic/stable/Fedora-Atomic-26-20170905.0/CloudImages/x86_64/images/Fedora-Atomic-26-20170905.0.x86_64.qcow2',
              'centos6': 'https://cloud.centos.org/centos/6/images/CentOS-6-x86_64-GenericCloud.qcow2',
              'centos7': 'https://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud.qcow2',
              'centos7atomic': 'http://cloud.centos.org/centos/7/atomic/images/CentOS-Atomic-Host-7-GenericCloud.qcow2',
              'cirros': 'https://download.cirros-cloud.net/0.3.5/cirros-0.3.5-x86_64-disk.img',
-             'cloudforms': 'https://access.redhat.com/downloads/content/167/ver=/cf-me---4.2/4.2/x86_64/product-software',
+             'cloudforms': 'https://access.redhat.com/downloads/content/167/ver=/cf-me---4.5/4.5/x86_64/product-software',
```

Comment:
maybe makes more sense to keep old version and create cloudforms42 and cloudforms45